### DISCUSSION TITLE
Feature Request / Keethief

### DISCUSSION THREAD

**Discussion by rascapac** [Pennyw0rth/NetExec#427] [Ideas]
Hi guys, it would be nice to include along the keepass discover and trigger modules, a keethief module in order to retrieve keepass password

**Comment by NeffIsBack** [comment]
Would probably a nice idea, feel free to open up an issue with that so we can tag it as a feature request

**Reply by NeffIsBack** [reply]
And of course if you have the time, develop and PR a module :)
